PhysicsTools/Heppy/python/physicsobjects/Electron.py

```diff
@@ -565,29 +564,26 @@ def hltPFIso(self,isoType):
         else: isoValue = -999
         return max(0, isoValue - self.rhoHLT*hltEA)
 
-    def chargedHadronIsoR(self,R=0.4):
+    def chargedHadronIso(self,R=0.4):
```

Comment:
If you rename these functions, change also the references in 
https://github.com/CERN-PH-CMG/cmg-cmssw/blob/heppy_94X_dev/PhysicsTools/Heppy/python/analyzers/objects/LeptonAnalyzer.py#L454-L459 in order to avoid breaking the analyzer.

Reply:
and also:
 * https://github.com/CERN-PH-CMG/cmg-cmssw/blob/heppy_94X_dev/PhysicsTools/Heppy/python/analyzers/objects/autophobj.py#L89-L90 
 * https://github.com/CERN-PH-CMG/cmg-cmssw/blob/heppy_94X_dev/PhysicsTools/Heppy/python/physicsobjects/Electron.py#L643-L661
 * https://github.com/CERN-PH-CMG/cmg-cmssw/blob/heppy_94X_dev/PhysicsTools/Heppy/python/physicsobjects/Muon.py#L148-L166